tests/nataliia_romanchuk/homework_4/test_uncompleted_comprehensions.py

```diff
@@ -0,0 +1,86 @@
+from itertools import tee
+
+# comprehensions
+some_list = [1, 2, 3, 4, 5]
+some_dict = {"a": 1, "b": 2, "c": 3}
+
+
+def test_comprehensions():
+    # generator expression
+    generator_result = list((i for i in some_list))
```

Comment:
you can avoid parenthesis if generator expression is only parameter

    generator_result = list(i for i in some_list)